tests/unit/python/foglamp/services/core/api/test_common_ping.py

```diff
@@ -76,40 +122,144 @@ def ssl_ctx(certs_path):
+                {"value": 6, "key": "SENT_4", "description": "blah5"},
+               ]}
+
+    mockedStorageClient = MagicMock(StorageClient)
+    with patch.object(middleware._logger, 'info') as logger_info:
+        with patch.object(connect, 'get_storage', return_value=mockedStorageClient):
+            with patch.object(mockedStorageClient, 'query_tbl_with_payload', return_value=result) as query_patch:
+                app = web.Application(loop=loop, middlewares=[middleware.optional_auth_middleware])
+                # fill route table
+                routes.setup(app)
+
+                server = await test_server(app, ssl=ssl_ctx)
+                server.start_server(loop=loop)
+
+                url = 'https://127.0.0.1:{}/foglamp/ping'.format(server.port)
+                async with aiohttp.ClientSession(loop=loop) as session:
+                    async with session.get(url, verify_ssl=False) as resp:
+                        assert 200 == resp.status
+                        content = await resp.text()
+                        content_dict = json.loads(content)
+                        assert 0.0 < content_dict["uptime"]
+
+                with pytest.raises(aiohttp.ClientConnectorSSLError) as error_exec:
+                    client = await test_client(server)
+                    resp = await client.get('/foglamp/ping')
+                assert "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed" in str(error_exec)
+
+                with pytest.raises(aiohttp.ClientConnectorSSLError) as error_exec:
+                    # self signed certificate,
+                    # and we are not using SSL context here for client as verifier
+                    connector = aiohttp.TCPConnector(verify_ssl=True, loop=loop)
+                    client = await test_client(server, connector=connector)
+                    resp = await client.get('/foglamp/ping')
+                assert "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed" in str(error_exec)
+
+                connector = aiohttp.TCPConnector(verify_ssl=False, loop=loop)
+                client = await test_client(server, connector=connector)
+                resp = await client.get('/foglamp/ping')
+                s = resp.request_info.url.human_repr()
+                assert "https" == s[:5]
+                assert 200 == resp.status
+                content = await resp.text()
+                content_dict = json.loads(content)
+                assert 0.0 < content_dict["uptime"]
+                assert 2 == content_dict["dataRead"]
+                assert 18 == content_dict["dataSent"]
+                assert 1 == content_dict["dataPurged"]
+                assert content_dict["authenticationOptional"] is True
+                calls = [call('Received %s request for %s', 'GET', '/foglamp/ping'),
+                         call('Received %s request for %s', 'GET', '/foglamp/ping')]
+                logger_info.assert_has_calls(calls)
```

Comment:
Please add `query_patch.assert_called_once_with(payload)` and fix indentation for logger_ info